pkg/controller/controller_test.go

```diff
@@ -437,7 +437,7 @@ func (s *ControllerSuite) TestExecActionSet(c *C) {
 
 func (s *ControllerSuite) TestRuntimeObjEventLogs(c *C) {
 	c.Skip("This may not work in MiniKube")
-	ctx := context.TODO()
+	contxt := context.TODO()
```

Comment:
```suggestion
	ctx := context.TODO()
```

Reply:
I don't think we should do that, `ctx` is being initialized later in the function with `context.Background` in line no 468.

Reply:
so why don't we use this one there as well.